pyhf/tensor/mxnet_backend.py

```diff
@@ -0,0 +1,183 @@
+import mxnet as mx
+from mxnet import nd
+import logging
+log = logging.getLogger(__name__)
+
+
+class mxnet_backend(object):
+    """Backend for MXNet"""
+
+    def __init__(self, **kwargs):
+        self.session = kwargs.get('session')
```

Comment:
seems like MXNet does not have a concept of a session (at least I can't find a reference to it. So I think this can be removed?

Reply:
Yeah, that was the result of some sloppy copypasta and me not checking things late at night. I'll fix that up when I get back to this after finishing work tonight.